Riot/Modules/SetPinCode/EnterPinCode/EnterPinCodeViewController.storyboard

```diff
@@ -252,15 +253,18 @@
                                             <stackView opaque="NO" contentMode="scaleToFill" distribution="fillEqually" alignment="center" spacing="20" translatesAutoresizingMaskIntoConstraints="NO" id="Nrp-tS-u1k">
                                                 <rect key="frame" x="0.0" y="216" width="220" height="60"/>
                                                 <subviews>
-                                                    <button opaque="NO" userInteractionEnabled="NO" tag="-99" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="DEv-rc-fGB">
+                                                    <button opaque="NO" userInteractionEnabled="NO" tag="-99" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="DEv-rc-fGB">
                                                         <rect key="frame" x="0.0" y="0.0" width="60" height="60"/>
+                                                        <accessibility key="accessibilityConfiguration">
+                                                            <bool key="isElement" value="NO"/>
+                                                        </accessibility>
```

Comment:
This is the only relevant change in the storyboard, all the other changes have been automatically generated by Xcode.